frontend/.vscode/settings.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
Not sure if we should check in this file?

Reply:
I see 3 options:
1. Enable automatic code fixing for all developers. In this case, leave the file as-is.
2. Disable automatic code fixing, i.e. remove this file.
3. A hybrid approach like the Angular devs use it: Rename the file to "settings-recommended.json" (or something else, that does not matter). And then just describe in the README file how to use it. See https://github.com/angular/angular/tree/main/.vscode

What do you think? I personally am not against checking in such configs, but if there are devs who want to work in a different way, then this can of course hinder them.

Reply:
I'd go for option 3. - either as a file or into a readme.